.github/workflows/ci.yml

```diff
@@ -22,7 +22,13 @@ jobs:
 
       - name: Setup
         run: |
-          build/jag setup
+          build/jag setup --skip-assets
+          SDK_PATH="$(build/jag setup --print-sdk-path)"
+          ASSETS_PATH="$(build/jag setup --print-assets-path)"
+          mkdir -p "$ASSETS_PATH"
+          export PATH="$PATH:$SDK_PATH/bin"
```

Comment:
Do we need this? You would think that passing `SDK_PATH` to `make assets` is enough. 

Which environment is changed by the `export`? Does it influence other CI steps?

Reply:
removed.
It wouldn't have affected later steps, as it wasn't `>> $GITHUB_PATH`.

done.